python/pyspark/errors/exceptions/connect.py

```diff
@@ -46,40 +46,56 @@ class SparkConnectException(PySparkException):
 
 def convert_exception(info: "ErrorInfo", message: str) -> SparkConnectException:
     classes = []
+    error_class = None
+    message_parameters = None
+    # TODO: add sqlState in PySparkException.
```

Comment:
I believe that since `PySparkException` manages errors that are specifically raised by PySpark (not JVM), maybe there is no need to add `sqlState` to it?
Btw the SQLSTATE of errors that occur in the JVM already can be obtained through the `getSqlState`.